source/_posts/2015-09-13-home-assistant-meets-ifttt.markdown

```diff
@@ -61,14 +61,17 @@ Setup Maker channel with event name `HA_FIRE_ALARM` and Twitter channel to tweet
 ```yaml
 # Configuration.yaml entry
 automation:
-  alias: Post a tweet when fire alarm is triggered
+- alias: Post a tweet when fire alarm is triggered
+  trigger:
+   - platform: mqtt
+     mqtt_topic: home/alarm/fire
+     mqtt_payload: 'on'
 
-  platform: mqtt
-  mqtt_topic: home/alarm/fire
-  mqtt_payload: 'on'
+  condition: use_trigger_values
```

Comment:
I wanted to show the condition field but I can easily remove. In this scenario, it would just be ignored anyway.

Which way do you prefer? I just wanted to update the post with the more relevant format.


Check out my virtualization blog at www.vCloudInfo.com<http://www.vcloudinfo.com>!
Follow me : Twitter: @ccostan | FaceBook.com<http://facebook.com>\VMwareInfo


On Sep 27, 2015, at 10:12 PM, Paulus Schoutsen <notifications@github.com<mailto:notifications@github.com>> wrote:


In source/_posts/2015-09-13-home-assistant-meets-ifttt.markdown<https://github.com/balloob/home-assistant.io/pull/84#discussion_r40514442>:

>
> -  platform: mqtt
> -  mqtt_topic: home/alarm/fire
> -  mqtt_payload: 'on'
> +  condition: use_trigger_values


There is no such thing as a MQTT condition.

-
Reply to this email directly or view it on GitHub<https://github.com/balloob/home-assistant.io/pull/84/files#r40514442>.